thirdparty/thorvg/src/common/tvgLock.h

```diff
@@ -38,10 +38,10 @@ namespace tvg {
     {
         Key* key = nullptr;
 
-        ScopedLock(Key& key)
+        ScopedLock(Key& p_key)
```

Comment:
@hermet FYI, I was getting a `-Wshadow` warning from GCC here.
Happy to make a PR upstream if this kind of change looks acceptable, possibly with a different naming convention if there's one you favor.